tests/Integration/Behaviour/Features/Context/CommonFeatureContext.php

```diff
@@ -111,18 +112,21 @@ public static function getContainer()
      */
     public static function clearDownloads(): void
     {
-        $filesToSkip = [
-            _PS_DOWNLOAD_DIR_ . 'index.php',
-            _PS_DOWNLOAD_DIR_ . '.htaccess',
-        ];
-
         foreach (glob(_PS_DOWNLOAD_DIR_ . '*') as $file) {
```

Comment:
Why not use `Filesystem` component instead? Since `_PS_DOWNLOAD_DIR_` is a dedicated test folder anyway you can remove its content recursively including folders

Reply:
Oh but I just noticed the folder does contain these two files `index.php` and `.htaccess` If tests can remove them maybe we should remove them from the repository as well no?

Reply:
Ah `.htaccess` is indeed removed, but not `index.php`

Reply:
index.php should also be deleted, we don't need it